pkg/sql/sem/tree/interval_test.go

```diff
@@ -374,8 +374,9 @@ func TestPGIntervalSyntax(t *testing.T) {
 		{`1yr`, types.IntervalTypeMetadata{}, `1 year`, ``},
 		{`1yrs`, types.IntervalTypeMetadata{}, `1 year`, ``},
 		{`1.5y`, types.IntervalTypeMetadata{}, `1 year 6 mons`, ``},
-		{`1.1y`, types.IntervalTypeMetadata{}, `1 year 1 mon 6 days`, ``},
-		{`1.11y`, types.IntervalTypeMetadata{}, `1 year 1 mon 9 days 14:24:00`, ``},
+		{`1.1y`, types.IntervalTypeMetadata{}, `1 year 1 mon`, ``},
```

Comment:
`math.Mod` can get tricky -- can you add tests for `-1.1y`?

Reply:
Added negative versions of all of these test cases and verified that it's the same in Postgres.